src/core.ts

```diff
@@ -15,7 +15,7 @@ const Inactive = require('pull-inactivity')
 const debug = require('debug')('secret-stack')
 
 function isPlainObject (o: unknown): o is Record<string, unknown> {
-  return o && typeof o === 'object' && !Array.isArray(o)
+  return !!o && typeof o === 'object' && !Array.isArray(o)
```

Comment:
right. This works a lot better for "", null & undefined :)

Reply:
It was also something that the new TypeScript compiler was asking me to do